app/forms/linking_case/search_form.rb

```diff
@@ -0,0 +1,34 @@
+module LinkingCase
```

Comment:
To follow the pattern for copy_case and inline with controller naming this module should be called `LinkCase`